### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by jmsche**
* Taskfile: user docker compose plugin instead of docker-compose
* Upgrade Symfony to 6.1.5 & other vendors
* Fix code style after php-cs-fixer upgrade
* Fix code & tests after EasyAdmin upgrade

### CODE REVIEW

File: src/Entity/S3Adapter.php [hunk 1]
```diff
@@ -10,10 +10,12 @@
 use App\Validator\Adapter;
 use Doctrine\DBAL\Types\Types;
 use Doctrine\ORM\Mapping as ORM;
+
 use function sprintf;
+
 use Symfony\Component\Validator\Constraints as Assert;
 
-#[Adapter(groups: ['Submit'])]
+#[Adapter(groups: ['eee'])]
```
Comment (bastien70) [inline src/Entity/S3Adapter.php:18]: Why 'eee' ? Take a look at S3AdapterCrudController in createNewForm() and createEditForm()

Comment (jmsche) [reply src/Entity/S3Adapter.php]: My bad, had to bypass the adapter for tests & forgot to rollback :)

**Comment by bastien70** [comment]
Thank you for the PR :)
Btw, did you upgraded recipes if needed ?

**Comment by jmsche** [comment]
FYI I had to switch from TextField to ChoiceField for the statusClass as unfortunately it failed with the TextConfigurator anymore.

**Comment by jmsche** [comment]
> Thank you for the PR :) Btw, did you upgraded recipes if needed ?

I'm going to, thanks for the reminder :D

**Comment by jmsche** [comment]
Done :+1:

**Review by bastien70 — APPROVED** [review]
Thank you ! :D

**Pull request merged**